### PR TITLE
Handle vendor already database with HIPAY_PROCESS = NO

### PR DESCRIPTION
**Pull request by etiennelandais**

### CODE REVIEW

File: src/Controller/LogVendorController.php [hunk 1]
```diff
@@ -65,11 +65,24 @@ protected function prepareAjaxData($data)
...
+            );
         }
 
         return $data;
     }
 
+    private function getEnableLabel($enable){
+        if($enable){
+            return $this->translator->trans("enabled");
+        }else{
```
Comment (etiennelandais) [inline src/Controller/LogVendorController.php:81]: MINOR Put one space between this "else" keyword and the opening curly brace. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)

File: src/Controller/LogVendorController.php [hunk 2]
```diff
@@ -65,11 +65,24 @@ protected function prepareAjaxData($data)
...
+            $data[$key]['enabled'] = array(
+                "enabled" => $logRow['enabled'],
+                "label" => $this->getEnableLabel($logRow['enabled'])
+            );
         }
 
         return $data;
     }
 
+    private function getEnableLabel($enable){
```
Comment (etiennelandais) [inline src/Controller/LogVendorController.php:78]: MINOR Move this open curly brace to the beginning of the next line. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)
MINOR Put one space between the closing parenthesis and the opening curly brace. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)

File: src/Controller/LogVendorController.php [hunk 3]
```diff
@@ -65,11 +65,24 @@ protected function prepareAjaxData($data)
...
+        if($enable){
```
Comment (etiennelandais) [inline src/Controller/LogVendorController.php:79]: MINOR Put one space between the closing parenthesis and the opening curly brace. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)
MINOR Put one space between this "if" keyword and the opening parenthesis. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)

File: src/Entity/OperationRepository.php [hunk 4]
```diff
@@ -48,6 +48,26 @@ public function save($operation)
     }
 
 
+    public function findVendorOperationsByPaymentVoucherId($operation){
```
Comment (etiennelandais) [inline src/Entity/OperationRepository.php:51]: MINOR Move this open curly brace to the beginning of the next line. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)
MINOR Put one space between the closing parenthesis and the opening curly brace. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)

**Comment by etiennelandais** [comment]
SonarQube analysis reported 7 issues
* ![MINOR][MINOR] 7 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by jprotin — APPROVED** [review]

**Pull request closed without merging**